mantidproject/mantid
# Setting attributes for composite functions.

Fixes #13743. 

**To test**
1. Open the multi-data set interface.
2. Add 2-3 spectra.
3. Add a composite fitting function.
4. Set its NumDeriv attribute.
5. Run fit and check that the attribute value was applied. (You can check history for that)
6. Try some other functions with attributes.
7. Check that they get passed to Fit.
